src/pomodoro/containers/PomodoroControls.js

```diff
@@ -74,8 +74,47 @@ class PomodoroControl extends React.PureComponent {
   }
 }
 
-const PomodoroControls = ({ status, play, pause, stop, skip }) => (
+
+class PomodoroShortcuts extends React.PureComponent {
+  componentDidMount() {
+    document.addEventListener("keydown", this.handleKeyboard)
```

Comment:
You can use single quotes here to follow the code style. Same for the cases on the switch statement.